include/deal.II/numerics/data_postprocessor.h

```diff
@@ -250,7 +250,7 @@ namespace DataPostprocessorInputs
      * or DataPostprocessorTensor may pass this flag to the constructor of
      * these three classes.
      */
-    std::vector<Tensor<1, spacedim>> solution_gradients;
+    std::vector<Tensor<1, spacedim, Number>> solution_gradients;
```

Comment:
This is `double` and I have the feeling that `DataPostprocessor` will convert any `number` to `double`. This wouldn't work for `std::complex`